apps/devportal/data/markdown/pages/learn/faq/xm-personalize/personalize-cdp.md

```diff
@@ -92,10 +84,9 @@ Regarding a full description, please refer to the documentation website. The fol
 **Real-Life Example (Jetstar Airways &amp; Sitecore)**
 
 - [https://www.sitecore.com/blog/personalization/the-skys-the-limit-for-jetstar-airways ](https://www.sitecore.com/blog/personalization/the-skys-the-limit-for-jetstar-airways%C2%A0)
-- [https://sitecore.showpad.biz/webapp2/results?query=vercel%20pitch&amp;scope=content,coaching&amp;slug=0857aedc7594214236e1967f7837fd27 ](https://sitecore.showpad.biz/webapp2/results?query=vercel%20pitch&scope=content,coaching&slug=0857aedc7594214236e1967f7837fd27%C2%A0)
 
 To solve a challenge of Create and act on insights from a single customer profile driven by an omnichannel experience strategy.**Jetstar** Use Sitecore Experience Manager as the core digital transformation platform, Sitecore CDP to activate customer insights and Sitecore Personalize deployed across multiple channels.

```

Comment:
There is a missing space here between the period "." and Jetstar.  Also `Use` is capitalized when I don't think it should be (following the word Jetstar.

Reply:
The whole paragraph reads awkwardly, but I've made the fixes you highlighted as part of the next push.

Reply:
I see what you mean.  Really that jetstart section should read (with the help of ChatGPT)

To address the complexity of deriving and implementing insights from a unified customer profile in an omnichannel experience strategy, Jetstar leverages Sitecore Experience Manager as the main platform for digital transformation. It also uses Sitecore CDP to activate customer insights and Sitecore Personalize across multiple channels.

Reply:
I took this but changed 'leverages' to 'uses'. Otherwise, let's call this done!